### PR TITLE
Svietlov hw3

### PR DESCRIPTION
**Pull request by Igaross**

### CODE REVIEW

File: Svietlov/homework3/area_of_circle.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import math
+rad = int(input("radius of the circle in cm "))
+print("area of the circle is ", math.pi * (rad ** 2), "cm")
```
Comment (katemalash) [inline Svietlov/homework3/area_of_circle.py:3]: Just clarifying: square centimeter, not centimeter

File: Svietlov/homework3/area_of_circle.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+print("perimeter of the circle is ", 2 * math.pi * rad, "cm")
```
Comment (katemalash) [inline Svietlov/homework3/area_of_circle.py:4]: Task is done good in general.

File: Svietlov/homework3/calc_for_bytes.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+mbytes_inp = int(input("enter number of megabytes"))
```
Comment (katemalash) [inline Svietlov/homework3/calc_for_bytes.py:1]: Information amount should be entered in bytes, not in megabytes.

File: Svietlov/homework3/collections.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# motorcycle marks collection
+moto_description = "motorcycles are fast like bullet"
+moto_marks = [moto_description, ["moto1", "moto2", "moto3", "moto4", "moto5"]]
+
+# elephants marks collection
+elph_description = "elephants are very big"
+elph_marks = [elph_description, ["elph1", "elph2", "elph3", "elph4", "elph5"]]
+
+
+# I think the best way to saving marks it`s lists
```
Comment (katemalash) [inline Svietlov/homework3/collections.py:16]: Are you sure? What about tuples?

File: Svietlov/homework3/collections.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+# at once, not individual marks.
+
+# THE SECOND PART
+
+# If I want to hold information only about unique marks,
+# I`m will use for this purposes type of string - set. For example:
+unique_description = "only unique marks"
+unique_collection = {"mark1", "mark2", "mark3", "mark4", "mark3", \
+     "mark1", "mark5"}
+print(unique_collection)
```
Comment (katemalash) [inline Svietlov/homework3/collections.py:27]: Good Decision, in general.

File: Svietlov/homework3/hw_no_temp.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+print("number switcher")
+a = int(input("enter you first number "))
+print("first number is ", a)
+b = int(input("enter your second number "))
+print("second number is ", b)
+a = -a - b
+b = -a - b
+a = -a - b
+print("a little magic")
+print("now first number is ", a, "and second number is ", b)
```
Comment (katemalash) [inline Svietlov/homework3/hw_no_temp.py:10]: Good Decision.

File: Svietlov/homework3/hypotenuse_of_triangle.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import math
+first_side = float(input("enter the first side of triangle "))
+second_side = float(input("enter the second side of triangle "))
+print("hypotenuse is ", math.sqrt (first_side ** 2 + second_side ** 2))
```
Comment (katemalash) [inline Svietlov/homework3/hypotenuse_of_triangle.py:4]: Good Decision.

File: Svietlov/homework3/perimetr_of_retangle.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+width_retangle = int(input("enter width in cm "))
+height_retangle = int(input("enter height in cm "))
+perimetr_retangle = 2 * (width_retangle + height_retangle)
+print("perimetr of retangle is ", perimetr_retangle, "cm")
```
Comment (katemalash) [inline Svietlov/homework3/perimetr_of_retangle.py:4]: Good Decision. And what about square?

**Review by katemalash — APPROVED** [review]
Home work is done good in general. You can merge pull-request after fixing some issues.

File: Svietlov/homework3/area_of_circle.py (hunk 1 above)
Comment (Igaross) [reply Svietlov/homework3/area_of_circle.py:3]: I'm not sure if I understood correctly which centimeters you mean.

File: Svietlov/homework3/area_of_circle.py (hunk 2 above)
Comment (Igaross) [reply Svietlov/homework3/area_of_circle.py:4]: Thank you.

File: Svietlov/homework3/calc_for_bytes.py (hunk 3 above)
Comment (Igaross) [reply Svietlov/homework3/calc_for_bytes.py:1]: Fixed in next update.

File: Svietlov/homework3/collections.py (hunk 4 above)
Comment (Igaross) [reply Svietlov/homework3/collections.py:16]: Maybe I got excited about a best way. The nice thing about tuples is that they are immutable.

File: Svietlov/homework3/collections.py (hunk 5 above)
Comment (Igaross) [reply Svietlov/homework3/collections.py:27]: Thank you.

File: Svietlov/homework3/hw_no_temp.py (hunk 6 above)
Comment (Igaross) [reply Svietlov/homework3/hw_no_temp.py:10]: Thank you.

File: Svietlov/homework3/hypotenuse_of_triangle.py (hunk 7 above)
Comment (Igaross) [reply Svietlov/homework3/hypotenuse_of_triangle.py:4]: Thank you.

File: Svietlov/homework3/perimetr_of_retangle.py (hunk 8 above)
Comment (Igaross) [reply Svietlov/homework3/perimetr_of_retangle.py:4]: Do you mean perimetr of square?  I'm not sure I understood.

**Pull request merged**